Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -228,8 +228,12 @@ MemoryViewWidget::MemoryViewWidget(Core::System& system, QWidget* parent)
           &MemoryViewWidget::UpdateBreakpointTags);
   connect(&Settings::Instance(), &Settings::EmulationStateChanged, this, [this] {
```

Comment:
While you're at it:
```suggestion
  connect(&Settings::Instance(), &Settings::EmulationStateChanged, this, [this] (const Core::State state) {
```
`EmulationStateChanged` has a `State` parameter so we don't need to call `Core::GetState` below.